src/primitives/sinsemilla.rs

```diff
@@ -1,7 +1,7 @@
 //! The Sinsemilla hash function and commitment scheme.
 
 use group::{Curve, Wnaf};
-use halo2::arithmetic::{CurveAffine, CurveExt};
+use halo2_proofs::arithmetic::{CurveAffine, CurveExt};
```

Comment:
`src/primitives/sinsemilla*` should be deleted; it is provided by `halo2_gadgets` now (since we _need_ the implementation over there).